src/stylesheets/components/_banner.scss

```diff
@@ -1,160 +1,171 @@
-    @include padding(0 null);
+  @include at-media('mobile-lg') {
+    @include u-padding-y(2px);
   }
 
   p {
-    @include margin(0 null);
-    display: inline;
-    overflow: hidden;
-    vertical-align: middle;
+    @include u-line-height($banner-font-family, 1);
+    @include u-margin-bottom(0);
+    @include u-margin-top(2px);
+    display: block;
+    overflow: visible;
 
-    @include media($small-screen) {
-      @include margin(2px null);
+    @include at-media('mobile-lg') {
+      @include u-margin-top(0);
       display: inline-block;
     }
   }
 
   img {
     float: left;
-    margin-right: 1rem;
+    margin-right: units(1);
     margin-top: 1px;
-    width: 2.4rem;
+    width: units(3);
 
-    @include media($small-screen) {
-      margin-right: 0.7rem;
-      width: 2rem;
+    @include at-media('mobile-lg') {
+      margin-right: units(1);
+      width: units(2.5);
     }
   }
 }
 
 .usa-banner-header-expanded {
-  align-items: center;
-  border-bottom: 1px solid $color-gray-light;
-  display: flex;
-  font-size: $h5-font-size;
-  min-height: 5.6rem;
-  padding-right: 3rem;
-
-  @include media($small-screen) {
+  @include u-font-size($banner-font-family, $theme-h5-font-size);
+  border-bottom: 1px solid $base-light;
+  min-height: units(7);
+  padding-right: units(4);
+
+  @include at-media('mobile-lg') {
+    @include u-font-size($banner-font-family, 1);
     border-bottom: none;
     display: block;
-    font-size: 1.2rem;
-    font-weight: $font-normal;
     min-height: 0;
     padding-right: 0;
   }
 
   > .usa-banner-inner {
     margin-left: 0;
 
-    @include media($small-screen) {
+    @include at-media('mobile-lg') {
       margin-left: auto;
     }
   }
 
   img {
-    margin-right: 2.4rem;
+    margin-right: units(3);
 
-    @include media($small-screen) {
-      margin-right: 0.7rem;
+    @include at-media('mobile-lg') {
+      margin-right: units(1);
     }
   }
 
   p {
-    line-height: $heading-line-height;
+    @include u-line-height($banner-font-family, $theme-heading-line-height);
     vertical-align: top;
 
-    @include media($small-screen) {
-      line-height: $base-line-height;
+    @include at-media('mobile-lg') {
+      @include u-line-height($banner-font-family, $theme-base-line-height);
       vertical-align: middle;
     }
   }
 }
 
 .usa-banner-button {
   @include button-unstyled;
-  @include padding(1.3rem null null 4.8rem);
+  @include u-padding-top(1.5);
+  @include u-padding-left(6);
+  @include u-font-size($banner-font-family, 1);
   background-position-x: right;
-  color: $color-primary;
+  color: $primary;
   display: block;
-  font-size: 1.2rem;
-  height: 4.3rem;
+  height: units(5);
   left: 0;
   position: absolute;
   text-decoration: underline;
   top: 0;
   width: 100%;
 
-  @include media($small-screen) {
-    @include padding(0 null null 0);
+  @include at-media('mobile-lg') {
+    @include u-padding-top(0);
+    @include u-padding-left(2px);
+    @include u-margin-left(.5);
```

Comment:
Add a leading 0 here, since that's what we follow in our Sass coding styleguide and show apply to mixins as well.